src/test/java/seedu/address/storage/JsonAdaptedMeetingTest.java

```diff
@@ -1,21 +1,19 @@
 package seedu.address.storage;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static seedu.address.storage.JsonAdaptedPerson.MISSING_FIELD_MESSAGE_FORMAT;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DATETIME;
 import static seedu.address.testutil.Assert.assertThrows;
 import static seedu.address.testutil.TypicalPersons.JAMAL;
 
-import java.util.ArrayList;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
 import java.util.List;
 import java.util.stream.Collectors;
 
 import org.junit.jupiter.api.Test;
 
 import seedu.address.commons.exceptions.IllegalValueException;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Phone;
+import seedu.address.testutil.MeetingBuilder;
 
 public class JsonAdaptedMeetingTest {
     private static final String INVALID_NAME = "R@chel";
```

Comment:
Maybe this string could be removed if not in use?

Reply:
yup removed!